annotation-processing/src/test/resources/test/fields/setter/InvalidStudent.java

```diff
@@ -3,15 +3,13 @@
 import com.exactpro.epfast.annotations.FastField;
 import com.exactpro.epfast.annotations.FastType;
 
-import java.util.List;
-
 @FastType
 public class InvalidStudent {
 
     private String name;
 
     @FastField(name = "name")
-    public void SetName(String name) {
+    public void setName(String name) {
```

Comment:
Wasn't it made intentionally invalid for test purposes? If yes, then checkstyle has ability to switch checks for several lines. I think it's done with special comments and probably additional module